nemo/service/__init__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Can we move the service folder under deploy please? (if we can make the REST API work)

Reply:
Yes its been moved.